### PR TITLE
{bio}[GCCcore/12.2.0] Restrander v20230713

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/Restrander/Restrander-20230713-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    'config',
+    'figures',
+    'LICENSE',
+]
+
+sanity_check_paths = {
+    'files': ['bin/restrander', 'README.md'],
+    'dirs': ['config', 'figures'],
+}
+
```
Comment (smoors) [inline easybuild/easyconfigs/r/Restrander/Restrander-20230713-GCCcore-12.2.0.eb:36]: can you add a `sanity_check_commands` here? for example
```
sanity_check_commands = ['restrander --help']
```

Comment (verdurin) [reply easybuild/easyconfigs/r/Restrander/Restrander-20230713-GCCcore-12.2.0.eb:36]: Done.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Review by smoors — APPROVED** [review]
lgtm

**Pull request merged**